### PR TITLE
Proof of concept for RN Tests

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

File: __tests__/utils.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { stringToDeviceIDType, DeviceIdType } = require("../Utils.js");
```
Comment (turtledreams) [inline __tests__/utils.test.js:1]: add more comments how purpose of the test etc

File: Utils.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ function configToJson(config) {
         }
         // APM END --------------------------------------------
         if (config.disableAdditionalIntentRedirectionChecks) {
-            json['disableAdditionalIntentRedirectionChecks'] = config.disableAdditionalIntentRedirectionChecks;
+            json.disableAdditionalIntentRedirectionChecks = config.disableAdditionalIntentRedirectionChecks;
```
Comment (turtledreams) [inline Utils.js:99]: maybe you can return this back. dunno why prettier jumped on to this

File: jest.config.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    collectCoverage: true,
+    collectCoverageFrom: [
+        "**/*.{js,jsx}",
+        "!**/node_modules/**",
+        "!**/vendor/**",
+        "!**/example/**",
+    ],
+    testPathIgnorePatterns: ["/node_modules/", "AwesomeProject"],
+    transformIgnorePatterns: [
+        "node_modules/(?!((jest-)?react-native|@react-native(-community)?)|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|react-native-svg)"
```
Comment (turtledreams) [inline jest.config.js:18]: with these options things work

**Review by turtledreams — CHANGES_REQUESTED** [review]
please add a readme inside __tests__ telling how to run tests and what current tests covers

File: __tests__/utils.test.js (hunk 1 above)
Comment (peterBrxwn) [reply __tests__/utils.test.js:2]: done

File: Utils.js (hunk 2 above)
Comment (peterBrxwn) [reply Utils.js]: done

File: jest.config.js (hunk 3 above)
Comment (peterBrxwn) [reply jest.config.js:18]: awesome!

**Pull request merged**